draft-ietf-quic-transport.md

```diff
@@ -5550,7 +5554,7 @@ The initial contents of this registry are tabulated in {{frame-types}}.
 IANA \[SHALL add/has added] a registry for "QUIC Transport Error Codes" under a
 "QUIC Protocol" heading.
 
-The "QUIC Transport Error Codes" registry governs a 16-bit space.  This space is
+The "QUIC Transport Error Codes" registry governs a 62-bit space.  This space is
 split into two spaces that are governed by different policies.  Values with the
```

Comment:
I think that we want to crib from the frame types (or other varint registries) when it comes to policies.  There isn't really a "first byte" any more.

Reply:
Thanks, done.